src/analysis/processing/qgsalgorithmcategorizeusingstyle.cpp

```diff
@@ -234,7 +234,7 @@ QVariantMap QgsCategorizeUsingStyleAlgorithm::processAlgorithm( const QVariantMa
 
   if ( matched > 0 )
   {
-    feedback->pushInfo( QObject::tr( "Matched %1 categories to symbols from file." ).arg( matched ) );
+    feedback->pushInfo( QObject::tr( "Matched %n categorie(s) to symbols from file.", nullptr, matched ) );
```

Comment:
```suggestion
    feedback->pushInfo( QObject::tr( "Matched %n categories to symbols from file.", nullptr, matched ) );
```

I think this is the correct way to handle this type of string (ping @m-kuhn )

Reply:
Yes, that looks perfect.